### PR TITLE
add mock data #7

### PR DESCRIPTION
**Pull request by pixelPepe**

### CODE REVIEW

**Review by nadyasav — APPROVED** [review]
Changes approved

File: frontend/src/data/mockAvailablePools.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "address": "0x4681de1e080e889fad73853fa7adaa0be39e433f",
+    "date":1696954119482,
+    "anchorCurrency" : "0x6265617665726275696c642e6f7267",
+    "tokens": ["0x6265617665726275696c642e6f7267","0x6265617665726275696c642e6f7267", "0x6265617665726275696c642e6f7267"],
+    "operatorFee": 12,
+    "total": 234500,
```
Comment (utgarda) [inline frontend/src/data/mockAvailablePools.json:8]: Total wut?

File: frontend/src/data/mockAvailablePools.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "profitPercent": 10,
```
Comment (utgarda) [inline frontend/src/data/mockAvailablePools.json:9]: Whose profit percent that may be?

File: frontend/src/data/mockAvailablePools.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    "profit": 3453453
```
Comment (utgarda) [inline frontend/src/data/mockAvailablePools.json:10]: Probably the pool's `totalProfit`

File: frontend/src/data/mockAvailablePools.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    "address": "0x4681de1e080e889fad73853fa7adaa0be39e433f",
+    "date":1696954119482,
+    "anchorCurrency" : "0x6265617665726275696c642e6f7267",
+    "tokens": ["0x6265617665726275696c642e6f7267","0x6265617665726275696c642e6f7267", "0x6265617665726275696c642e6f7267"],
+    "operatorFee": 12,
+    "total": 234500,
+    "profitPercent": 10,
+    "profit" : 30000
+  }
+]
```
Comment (utgarda) [inline frontend/src/data/mockAvailablePools.json:32]: endline :)

**Pull request merged**